packages/themes/default/styles/variables/_navbar.scss

```diff
@@ -0,0 +1,115 @@
+/* Navbar */
+
+//  Spacing
+$theme-site-navbar-padding-x:           16px !default;
+$theme-site-navbar-padding-y:           0 !default;
+$theme-site-navbar-brand-margin-y: 0 !default;
+$theme-site-navbar-brand-margin-x: 12px !default;
+
+// Colors
+
+// Dark
+$theme-site-navbar-dark-color:          rgba($white, .75) !default;
+$theme-site-navbar-dark-hover-color:    $white !default;
+$theme-site-navbar-dark-active-color:   $theme-site-navbar-dark-hover-color !default;
+
+// Light
+$theme-site-navbar-light-color:         rgba($black, .75) !default;
+$theme-site-navbar-light-hover-color:   $black !default;
+$theme-site-navbar-light-active-color:  $theme-site-navbar-light-hover-color !default;
+
+// Themes
+$theme-site-navbar-map: (
```

Comment:
The map will later be used for more than just colors, hence the use of a more broad terminology at this time.

Reply:
I'm a little worried about using such a deep map for all of these values. Why not maintain individual variables like before?

Reply:
Also, the mixin is now dependent on the map variable and its structure. Previously, the mixin could be use with _any_ color values (like BizBash was doing)

Reply:
Yes, the mixin is now dependent upon having a map passed in; however, the functionality of it has not changed, beyond the fact that it can now support nested maps--which reduces the likelihood of duplicate code as it keeps the definitions for a component all housed within the same area. This will become especially useful in future stages, as the same mixin can be utilized to pull in more than just link color assignments.

Here's an example of where this map can lead:

```
// Colors

// Dark
$theme-site-navbar-dark-color:          rgba($white, .75) !default;
$theme-site-navbar-dark-hover-color:    $white !default;
$theme-site-navbar-dark-active-color:   $theme-site-navbar-dark-hover-color !default;
$theme-site-navbar-dark-bg-color:       $dark !default;

// Light
$theme-site-navbar-light-color:         rgba($black, .75) !default;
$theme-site-navbar-light-hover-color:   $black !default;
$theme-site-navbar-light-active-color:  $theme-site-navbar-light-hover-color !default;

// Lighter
$theme-site-navbar-lighter-color:         $black !default;
$theme-site-navbar-lighter-hover-color:   $primary !default;
$theme-site-navbar-lighter-active-color:  $theme-site-navbar-lighter-hover-color !default;
$theme-site-navbar-lighter-bg-color:      $primary !default;

// Fonts
$theme-site-navbar-primary-font-size:   16px !default;
$theme-site-navbar-secondary-font-size: 14px !default;

// Decorations
// Notes: 
// Primary/secondary both utilize the same settings currently
// Best to combine them into a single pull
$theme-site-navbar-link-decoration:         none !default;
$theme-site-navbar-link-hover-decoration:   underline !default;
$theme-site-navbar-link-active-decoration:  $theme-site-navbar-link-hover-decoration !default;

// Themes
$theme-site-navbar-map: (
  dark: (
    default:    $theme-site-navbar-dark-color,
    hover:      $theme-site-navbar-dark-hover-color,
    active:     $theme-site-navbar-dark-active-color,
    background: $theme-site-navbar-dark-bg-color
  ),
  light: (
    default:    $theme-site-navbar-light-color,
    hover:      $theme-site-navbar-light-hover-color,
    active:     $theme-site-navbar-light-active-color
  ),
  lighter: (
    default:    $theme-site-navbar-lighter-color,
    hover:      $theme-site-navbar-lighter-hover-color,
    active:     $theme-site-navbar-lighter-active-color,
    background: $theme-site-navbar-lighter-bg-color
  ),
  fonts: (
    primary:    $theme-site-navbar-primary-font-size,
    secondary:  $theme-site-navbar-secondary-font-size
  ),
  decorations: (
    default:    $theme-site-navbar-link-decoration,
    hover:      $theme-site-navbar-link-hover-decoration,
    active:     $theme-site-navbar-link-active-decoration
  )
);
```